tests/integration/adapter-node-tests.js

```diff
@@ -2,6 +2,7 @@
 
 export default function adapterNodeTests() {
   it('should handle recording requests posting a Buffer', async function() {
+    expect(false).to.equal(true);
```

Comment:
Remove?

Reply:
This is demonstrating that this test doesn’t appear to be running (even on master) (see merge blocker list)